examples/gpt2_inference_server.py

```diff
@@ -45,6 +46,7 @@ def parse_args():
         default="cache_hf",
         help="cache dir for huggingface objects",
     )
+    parser.add_argument("--async_server", action="store_true")
```

Comment:
`llama_training.py` and `gpt2_training.py` follow `--long-arg-name` style. Please update to the style